test/test-fs.c

```diff
@@ -2453,6 +2453,49 @@ TEST_IMPL(fs_non_symlink_reparse_point) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+TEST_IMPL(fs_lstat_windows_store_apps) {
+  uv_loop_t* loop;
+  char localappdata[MAX_PATH];
+  char windowsapps_path[MAX_PATH];
+  char file_path[MAX_PATH];
+  size_t len;
+  int r;
+  uv_fs_t req, stat_req;
+  uv_dirent_t dirent;
+
+  loop = uv_default_loop();
+  ASSERT(loop != NULL);
+  len = MAX_PATH;
+  r = uv_os_getenv("LOCALAPPDATA", localappdata, &len);
+  if (r == UV_ENOENT) {
+    return TEST_SKIP;
+  }
+  ASSERT(r == 0);
+  r = sprintf_s(windowsapps_path,
```

Comment:
Just curious, is there a reason to prefer `sprintf_s()` over `snprintf()`?

Reply:
TBH, IDK. I guess out of habit from some other project? I'll change those to `snprintf` to match rest of the codebase.